components/Footer/index.tsx

```diff
@@ -139,11 +139,11 @@ function Footer() {
                         // @info why the router here does not work? URL update but UI not: because when we're pushing to the
                         // same page e.g /ilhamb to /ssiprotocol it'll not trigger useeffect (but if from ilhamb/didx to /ssiprotocol this is works)
                         // Router.push('/ssiprotocol/tree')
-                        window.open('http://tyron.network/ssiprotocol', '_self')
+                        // window.open('http://tyron.network/ssiprotocol', '_self')
```

Comment:
why commented out?

Reply:
Last time I disable this so Koni can do testing and inspect the resolved addr. Fixed, I just put it back